NOTICE.txt

```diff
@@ -2,7 +2,7 @@
                            Project Helidon
                            ===============
 
-Copyright (c) 2017, 2019 Oracle and/or its affiliates. All rights reserved.
+Copyright (c) 2017, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
We can (are supposed to?) remove "All rights reserved" from the copyright notices now.

Reply:
Fixed.